pkgs/development/python-modules/pysonic/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "py-sonic";
+  version = "0.7.7";
+  disabled = pythonOlder "3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4cea42a2b0dc2ed0fd8568d6bf0509cfa2675a8b1c347ce9364a00881ebc0272";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = https://github.com/crustymonkey/py-sonic;
```

Comment:
URLs need to be quoted per https://github.com/NixOS/rfcs/pull/45.

Reply:
Fixed